frontend/controller/actions/index.js

```diff
@@ -191,8 +187,9 @@ sbp('sbp/selectors/register', {
         contractID,
         contractName,
         data: [
-          ...keyShares.map((data) => ['chelonia/out/keyShare', { data: encryptedOutgoingData(contractID, CEKid, data) }]),
-          ['chelonia/out/keyUpdate', { data: updatedKeys }]
+          ...(keyShares[0] ?? []),
+          ['chelonia/out/keyUpdate', { data: updatedKeys }],
+          ...(keyShares[1] ?? [])
```

Comment:
I'm a little confused as to the nature of the "arbitrary operations" you say are supported here, as this seems to support at most 2 operations (`keyShares[0]` and `keyShares[1]`). What if there are 3?

Reply:
The `keyShares` name is perhaps confusing, and I'll update it accordingly, same with your suggestion for explaining the signature. That said, it really does support arbitrary operations, in any quantity (see the spread operator). `keyShares[0]` is what comes before `keyUpdate` and `keyShares[1]` is what comes after. A hypothetical `keyShares[2]` doesn't make sense.